src/api/v1/middlewares/validation/userSignup.js

```diff
@@ -22,11 +22,15 @@ export const validateSignup = async (req, res, next) => {
 
 	const { error } = schema.validate(req.body);
 
+	const { password } = req.body;
 	if (error) {
 		return res.status(422).json({
 			error: `Please enter the ${error.details[0].context.key} of maximum 40`,
 		});
 	}
 
+	const hashedPassword = await bcrypt.hash(password, 10);
```

Comment:
The hashing of the password should be done on the UserModel where database/business logic stays.